acceptable/openapi_exporter.py

```diff
@@ -0,0 +1,45 @@
+"""
+Helpers to translate acceptable metadata to OpenAPI specifications (OAS).
+"""
+
+import sys
+from typing import Any
+
+import yaml
+
+
+def _to_dict(source: Any):
+
+    if hasattr(source, "_to_dict"):
+        return source._to_dict()  # noqa
+    elif type(source) == dict:
+        source_dict = {}
+        for key, value in source.items():
+            source_dict[key] = _to_dict(value)
+        return source_dict
+    elif type(source) == list:
+        source_list = []
+        for item in source:
+            source_list.append(_to_dict(item))
+        return source_list
+    else:
+        return source
+
+
+def dump_oas(schema, stream):
+
+    return yaml.safe_dump(
+        _to_dict(schema),
+        stream,
+        default_flow_style=False,
+        encoding=None,
+    )
+
+
+def print_oas(schema):
+    dump_oas(schema, sys.stdout)
```

Comment:
I wonder if we really need a single-line function for printing to stdout. And another similar one for writing to a file. I see `dump_oas()` does expect a stream to write stuff to... Maybe it could return a string and let the caller print it or write to file? In that case I'd probably call it `dumps()` (`"_oas"` might be a bit redundant? or is it expected to "dump" something else? Hm well, it should be able to return openapi 3.0.x as well... but it could be an arg to `dump()`... ).

Oh, one more thing: `acceptable.openapi.dumps()` would look better imho :-D

(This is all kinda nitpicky! Feel free to ignore.)

Reply:
This is a vestigial debugging helper! I've removed it :)